### PR TITLE
exercises

### PR DESCRIPTION
**Pull request by eoneruw**

### CODE REVIEW

File: students/eoner/lesson02/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+def fizbuzz(l=100):
+    for i in range(l+1):
+        if i % 3 == 0 and i % 5 == 0:
+            print("FizzBuzz")
+        elif i % 3 == 0:
+            print("Fizz")
+        elif i % 5 == 0:
+            print("Buzz")
+        else:
+            print(i)
```
Comment (majstorpoki) [inline students/eoner/lesson02/FizzBuzz.py:15]: I'd appreciate if you also included testing, but this is still fine. I like the default argument used.

File: students/eoner/lesson02/grid.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def print_grid(x, s):
+    for i in range(x):
+        print((((("+" + ("-"*s)))*x) + "+")+(("\n"+((("|" + (" "*s)))*x) + "|")*s))
+    print(((("+" + ("-"*s)))*x) + "+")
+
+print_grid(3,4)
```
Comment (majstorpoki) [inline students/eoner/lesson02/grid.py:6]: You could include some comments, doc strings, etc. but this definitely does work. Thanks.

File: students/eoner/lesson02/series.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    # test if sum_series works for arbitrary initial values
+    assert sum_series(0, 3, 2) == 3
+    assert sum_series(1, 3, 2) == 2
+    assert sum_series(2, 3, 2) == 5
+    assert sum_series(3, 3, 2) == 7
+    assert sum_series(4, 3, 2) == 12
+    assert sum_series(5, 3, 2) == 19
+
+    print("tests passed")
```
Comment (majstorpoki) [inline students/eoner/lesson02/series.py:65]: Thanks

File: students/eoner/lesson03/strformat_lab.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from decimal import Decimal
```
Comment (majstorpoki) [inline students/eoner/lesson03/strformat_lab.py:1]: I like how you explored using Decimal. Thanks.

**Review by majstorpoki — COMMENTED** [review]
Thanks so much.

File: students/eoner/lesson03/strformat_lab.py (hunk 4 above)
Comment (eoneruw) [reply students/eoner/lesson03/strformat_lab.py:1]: thanks - pretty crude implementation

**Pull request merged**